platform/android/audio_driver_opensl.cpp

```diff
@@ -329,4 +356,14 @@ void AudioDriverOpenSL::set_pause(bool p_pause) {
 }
 
 AudioDriverOpenSL::AudioDriverOpenSL() {
+	recordItf = nullptr;
```

Comment:
I'd say these should be initialized in the header

Reply:
So, just set these member variables to `nullptr` right where they're declared in https://github.com/godotengine/godot/blob/master/platform/android/audio_driver_opensl.h#L57 ?

Reply:
Yep, that's how we do it elsewhere

Reply:
Should be addressed as of https://github.com/godotengine/godot/pull/85955/commits/0feca83b0745a3d2f48e9e5e75d00dcb74efa139